index.js

```diff
@@ -82,11 +100,47 @@ cm.Item({
   }
 });
 
+// Vine context menu on Vine.co
+// only works from direct embed URLs structured like this: 
+// https://vine.co/v/[vineID] or https://vine.co/v/[vineID]/embed
+cm.Item({
+  label: "Send to mini player",
+  context: [
+  	cm.PredicateContext(checkLocation),
+  	cm.SelectorContext('[src*="mediasource:https://vine.co/"]')
+  ],	
+  contentScript: "self.on('click', function (node, data) {" +
+                 "  self.postMessage(document.URL);" +
+                 "});",
+  onMessage: function(pageURL) {
+    updatePanel(constructVineEmbedUrl(pageURL));
+  }
+});
+
+function checkLocation(opts) {
+	if (opts.documentURL.indexOf('vine.co/v/') > 0) {
+		return true;
+	} else if (opts.documentURL.indexOf('twitter.com') > 0) {
```

Comment:
I think we just want to check that the url is shaped like `vine.co/v/` here right? In that case we would need to check both `hrefURL` and `srcURL`